src/node_dotenv.cc

```diff
@@ -17,9 +17,9 @@ using v8::String;
  * available at https://github.com/motdotla/dotenv
  */
 const std::regex LINE(
-    "\\s*(?:export\\s+)?([\\w.-]+)(?:\\s*=\\s*?|:\\s+?)(\\s*'(?:\\\\'|[^']"
-    ")*'|\\s*\"(?:\\\\\"|[^\"])*\"|\\s*`(?:\\\\`|[^`])*`|[^#\r\n]+)?\\s*(?"
-    ":#.*)?");  // NOLINT(whitespace/line_length)
+    "(?:^|^)\\s*(?:export\\s+)?([\\w.-]+)(?:\\s*=\\s*?|:\\s+?)(\\s*'(?"
```

Comment:
The previous implemention is copied from `dotenv` but the regexp is missing the beginning part `(?:^|^)`.

So, it cannot determine the commented env properly.

Reply:
Seems like it is removed because `std::regex::multiline` is not supported.
The current workflow using `C++11` only and that required `C++17`

Reply:
I'm not sure that `(?:^|^)` part is makes much sense, and can be simplified just to `(?:^)`

Reply:
After testing, it can be simplified to `(?:^)`.
Due the problem of `std::regex::multiline` not supported in some environment.
I believe the option of just changing the regex is a no-go.